src/AasxServerDB/EditDB.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AasCore.Aas3_0;
+using AdminShellNS;
+using Microsoft.EntityFrameworkCore;
+
+namespace AasxServerDB
+{
+    public class EditDB
+    {
+        static public void EditAAS(AdminShellPackageEnv env)
+        {
+            Dictionary<string, int> aasToDeleteAASXIdsDic; 
+
+            var aasList = env.AasEnv.AssetAdministrationShells;
+            var aasIds = aasList.Select(x => x.Id).ToList();
+
+            using (AasContext db = new AasContext())
+            {
+                // Delets automtically from DB

```

Comment:
```suggestion
                // Deletes automatically from DB

```

Please check your spelling